flixel/FlxG.hx

```diff
@@ -80,7 +80,12 @@ class FlxG
 	/**
 	 * How fast or slow time should pass in the game; default is `1.0`.
 	 */
-	public static var timeScale:Float = 1;
+	public static var timeScale:Float = 1.0;
```

Comment:
FYI 1.0 and 1 are functionally equivalent when typed to Float

Reply:
I figured so, however it was a change done mostly for comment accuracy, and also due to `FlxG.reset` setting these values to `1.0`.

Either way, should I revert this change?

Reply:
no need to revert, just saying that this change doesn't make it "more accurate" in any meaningful way